FAQ.md

```diff
@@ -25,9 +25,11 @@
 
 *   *What do the [labels](https://github.com/swcarpentry/lesson-template/issues?q=is%3Aopen+is%3Aissue) mean?*
 
+    *   `build`: changes to `Makefile` or lesson creation process
     *   `bug`: something is wrong in our tools or documentation
     *   `defer`: marks an issue that we will work on, but not now
     *   `discussion`: marks issues used for conversations about specific problems and questions
+    *   `documentation`: information about how to use the lesson-template
     *   `duplicate`: marks an issue that was closed as redundant (include the number of the original issue in the closing comment)
     *   `enhancement`: asks for, or adds, a new feature or new information
     *   `filed-by-newcomer`: issue or pull request was filed by someone who is relatively new to GitHub and/or this project,
```

Comment:
I would remove this `filed by newcomer`. It feels unwelcoming to me. I suggest using `suitable for newcomer`. I would also change `suitable for newcomer` to `newcomer-friendly`.